portfolio/src/main/webapp/script.js

```diff
@@ -40,7 +40,26 @@ function viewGame(){
 }
 
 function getGreeting() {
-  fetch('/data').then(response => response.text()).then((name) => {
+  fetch('/data')
+  .then(response => response.text())
+  .then((name) => {
     document.getElementById('greeting').innerText = name;
   });
+}
+
+function getJSONData(){
+  fetch('/data')  
+  .then(response => response.json()) 
+  .then((message) => {
+
+      let messageConntainer = document.getElementById("messages");
+      console.log(message[0]);
+
+      for(let i = 0; i < message.length; i++){
+        let text = document.createTextNode(message[i]);
```

Comment:
Do you want to attach this to a standard html tag: div, p?

Reply:
I did not specify where the text will go, but I will make it so it goes in a p tag in my next commit

Reply:
Thank you.